packages/tool_util/test-requirements.txt

```
@@ -1,3 +1,4 @@
 beaker
+sqlalchemy
```

This needs to be in sync with the pinning specified in `pyproject.toml`:

```suggestion
SQLAlchemy>=1.4.25,<2
```